src/frontend/main/MapView.tsx

```diff
@@ -0,0 +1,379 @@
+`;
+
+const FilterWrapper = styled.div<FilterWrapperProps>`
+  position: absolute;
+  top: 10px;
+  padding: ${props => (props.isEmbed ? '3px' : '3px 16px')};
+  display: flex;
+
+  @media (min-width: 624px) {
+    padding-left: 0;
+  }
+`;
+
+const ActiveFilter = styled.div`
+  height: 35px;
+  padding: 5px 16px;
+  border-radius: 100px;
+  font-weight: bold;
+  font-size: 16px;
+  background: ${props => props.theme.grey};
+  color: ${props => props.theme.white};
+`;
+
+const MapInfo = styled.div`
+  position: fixed;
+  bottom: 34px;
+  width: 100vw;
+  background: ${props => props.theme.white};
+  text-align: left;
+  border-top: 1px solid ${props => props.theme.black};
+  line-height: 1.25;
+
+  p {
+    margin: 5px 0;
+  }
+`;
+
+const MapInfoContent = styled(Container)`
+  padding: 6px 34px 0 16px;
+  position: relative;
+
+  @media (min-width: 624px) {
+    padding-left: 0;
+  }
+`;
+
+const TotalResponses = styled.div`
+  background: ${props => props.theme.white};
+  position: fixed;
+  bottom: 0px;
+  padding: 10px 0 10px 16px;
+  font-size: 14px;
+  font-style: italic;
+  width: 100vw;
+  font-weight: bold;
+  text-align: left;
+
+  p {
+    margin: 0;
+  }
+
+  @media (min-width: 624px) {
+    padding-left: 0;
+  }
+`;
+
+const CloseButton = styled.button`
+  cursor: pointer;
+  border: none;
+  background-color: transparent;
+  position: absolute;
+  top: 0;
+  right: 0;
+  z-index: 1;
+  padding: 16px;
+
+  svg {
+    width: 16px;
+    height: 16px;
+  }
+`;
+
+const MapView = (props: MapViewProps) => {
+  const currentPath = props.location!.pathname;
+  const isEmbed = currentPath === '/map-embed';
+  const { navHeight, headerHeight } = theme;
+  const topPartHeight = isEmbed ? navHeight : headerHeight + navHeight;
+  const [showMapInfo, setShowMapInfo] = useState(true);
+  const [selectedFilter, setSelectedFilter] = useState<FilterKey>(FILTERS.corona_suspicion_yes.id as FilterKey);
+  const [mapHeight, setMapHeight] = useState(window.innerHeight - topPartHeight - 10);
+  const data = props.responseData.data;
+  const [activeView, setActiveView] = useState<'MAP' | 'TABLE'>('MAP');
+
+  if (props.responseData === 'FETCHING') {
+    return <MessageContainer>Loading...</MessageContainer>;
+  }
+
+  if (props.responseData === 'ERROR') {
+    return <MessageContainer>Error loading data</MessageContainer>;
+  }
+
+  const cities: string[] = cartogramData
+    .sort((x: mapProperties, y: mapProperties) => d3.ascending(x.city, y.city))
+    .map((item: mapProperties) => {
+      return item.city;
+    });
+
+  const handleFilterChange = (filterName: string) => {
+    setSelectedFilter(filterName as FilterKey);
+  };
+
+  const totalResponses = data.reduce((accumulator: number, currentValue: any) => {
+    if (currentValue.responses < 0) return accumulator;
+    return accumulator + currentValue.responses;
+  }, 0);
+
+  window.addEventListener('resize', () => {
```

Comment:
Should this be put inside `useEffect` with call to `removeEventListener` in the cleanup function?

Reply:
Looks like this is fixed in master https://github.com/futurice/symptomradar/blob/master/src/frontend/main/map/MapView.tsx#L120 , probably after @Doniee 's route structure refactor? :thinking: 

Nice catch nevertheless :+1: 

I rebased master to resolve some merge conflicts and this file is moved to `src/frontend/main/map/MapView.tsx` now : D